### DISCUSSION TITLE
Tutorials for newbies?

### DISCUSSION THREAD

**Discussion by L00ncan** [mistic100/Photo-Sphere-Viewer#1125] [General]
Any tutorial for newbies, i'd like to install this on my bluehost webserver anyway possible, i don't really care how i install it as long as the finished product is installed i believe i can make my website glow with this, how do i install it? the same way i installed pannellum. Is this webserver friendly, bluehost won't throttle me for using this will they!, i'm looking for something less demanding and taxing for my webserver, but installing it is my main problem currently

**Comment by mistic100** [comment]
This is a Javascript library not a finished product, it cannot be installed through webhost control panels. Sorry but you will have to learn how to adapt it to your needs.

The documentation has a clear quick-start https://photo-sphere-viewer.js.org/guide/#your-first-viewer

**Comment by L00ncan** [comment]
Is there a future plans pathway of what i'm suggesting, i figured if it's on the web it's for easy web display like Pannellum, if not then... :(

**Reply by mistic100** [reply]
No there is not. I don't understand what you say about Pannellum, from what I see the install process is exatly the same as Photo Sphere Viewer https://pannellum.org/download/ : download some Javascript file and add it to your page.

I will provide help about using the library but I will not provide support if you don't know anything about web develloppement.

**Reply by L00ncan** [reply]
I was able to use Pannellum on my bluehost server very easily, i uploaded the folder to my webhost and was able to access my .jpgs very easily and directly. All i need to do is replace the .jpg with many of my hundreds of thousands of Panorama360s however i need options and hoping i can use this how i can use Pannellum, Pannellum was easy to install

https://cdn.pannellum.org/2.5/pannellum.htm#panorama=https://pannellum.org/images/jfk.jpg&hfov=120&autoRotate=-13&autoLoad=true

All i need to do is change the .jpg, i'm hoping to try other viewers i'm hoping i can install yours like i installed Pannellum but i'm not going to try unless even possible. I want to take my Panorama360s to the next level
